cowprotocol/cowswap
# 977/deprecate rinkeby

# Summary

Part of #977 

Disabling Rinkeby in the UI

This change is minimal and now the app respects the networks selected in the envvar `REACT_APP_SUPPORTED_CHAIN_IDS`

Full removal should be performed once the SDK removes rinkeby from the list of supported networks

  # To Test

1. Without connecting you wallet, observe the network selection
* It should not contain Rinkeby
2. Connect you wallet in a network other than rinkeby
* App should work as usual
3. Change the network in your wallet to rinkeby
* You should see the network unsupported error
